CVLByExamples/GeneralExamples/certora/spec/general.spec

```diff
@@ -0,0 +1,122 @@
+using DummyERC20A as erc20;
+
```

Comment:
comment:

/** General spec file that can used on any solidity contract, helps highlight the behavior or the contract **/